staticcheck.conf

```diff
@@ -1 +1,15 @@
-initialisms = ["inherit", "CNI", "IPAM"]
+initialisms = ["ACL", "API", "ASCII",
+	"CNI", "CPU", "CSS",
+	"DNS",
+	"EOF",
+	"GID", "GUID",
+	"HTML", "HTTP", "HTTPS",
+	"ID", "IP", "IPAM",
+	"JSON",
+	"QPS",
+	"RAM", "RPC",
+	"SLA", "SMTP", "SQL", "SSH",
+	"TCP", "TLS", "TTL",
+	"UDP", "UI", "UID", "URI", "URL", "UTF8", "UUID",
+	"VM",
+	"XML", "XMPP", "XSRF", "XSS"]
```

Comment:
please add NAT NATT IPSec, there are probably more, please wait for more suggestions, then we can propagate to other repos.